packages/eds-core-react/src/components/Datepicker/calendars/YearGrid.tsx

```diff
@@ -95,14 +103,20 @@ const GridFocusManager = ({
 export const YearGrid = ({
   setFocusedYear,
   year: selectedYear,
+  yearPickerPage,
 }: {
   setFocusedYear: (year: number) => void
   year: number
+  yearPickerPage: number
 }) => {
   return (
     <Grid>
       <FocusScope contain restoreFocus autoFocus>
```

Comment:
With the new interactive buttons in the header, this focusScope stuff probably needs to be changed, otherwise the page changer buttons are inaccessible with keyboard. Not sure how. Perhaps FocusScope can be moved out to encompass the entire popover